### PR TITLE
Convert dict_values object to list

### PR DESCRIPTION
**Pull request by hamstap85**
Recreation of https://github.com/jazzband/django-formtools/pull/154 with test

### CODE REVIEW

**Comment by hamstap85** [comment]
@claudep Here's the regression test

**Comment by hamstap85** [comment]
`coveralls` claims that coverage of a file untouched by this PR decreased???

**Comment by hramezani** [comment]
@hamstap85 Thanks. LGTM :+1:

File: tests/wizard/test_forms.py [hunk 1]
```diff
@@ -85,6 +85,12 @@ class TestWizardWithInitAttrs(TestWizard):
     instance_dict = {'start': User()}
 
 
+class TestWizardWithTypeCheck(TestWizard):
+    def done(self, form_list, **kwargs):
+        assert type(form_list) is list, "`form_list` was {}, should be a list".format(type(form_list))
+        return http.HttpResponse("All good")
```
Comment (claudep) [inline tests/wizard/test_forms.py]: I guess the coverage decrease is causing by the above code which will probably never run.

Comment (hamstap85) [reply tests/wizard/test_forms.py]: It should be running, `assert` will cause the test to fail, and it doesn't. The drop happens in `formtools/wizard/preview.py`

**Pull request merged**